src/groups/mqb/mqbcfg/mqbcfg_clusterquorummanager.h

```diff
@@ -0,0 +1,117 @@
+// Copyright 2019-2023 Bloomberg Finance L.P.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// mqbcfg_clusterquorummanager.h -*-C++-*-
+#ifndef INCLUDED_MQBCFG_CLUSTERQUORUMMANAGER
+#define INCLUDED_MQBCFG_CLUSTERQUORUMMANAGER
+
+/// @file mqbcfg_clusterquorummanager.h
+///
+/// @brief Provide a thread safe mechanism to manage cluster quorum.
+///
+/// @bbref{mqbcfg::ClusterQuorumManager} is a thread safe class that
+/// encapsulates the quorum value for a cluster, allowing it to be set and
+/// retrieved safely across multiple threads.
+
```

Comment:
```suggestion
///
/// Thread Safety
/// =============
///
/// This component is thread safe.

```
Makes it easier to check thread safefy